apps/accounts/serializers.py

```diff
@@ -69,3 +70,12 @@ def update(self, instance, validated_data):
             profile.linkedin_url = linkedin_url
             profile.save()
         return instance
+
+
+class CustomPasswordResetSerializer(PasswordResetSerializer):
```

Comment:
Rename  `CustomPasswordResetSerializer` to `PasswordResetSerializer`

Reply:
Done